conda/recipes/cuml/meta.yaml

```diff
@@ -38,7 +38,7 @@ requirements:
   run:
     - python x.x
     - cudf {{ minor_version }}
-    - dask_cudf {{ minor_version }}
+    - dask-cudf {{ minor_version }}
```

Comment:
Sorry if I'm just misunderstanding. Is this suppose to be `dask-cudf` or `dask-cuda`? Wasn't sure based on the commit message.

Reply:
I don't think us (or cuDF for that matter) have ever explicitly had dask-cuda as a dependency in the recipes, independent of this PR, do you think we should add it?

Reply:
It probably wouldn't hurt, but I'm sure you both know my suggestion would be for us to do it in a separate PR.

Reply:
Completely agree. Wasn't trying to hold this up. Just wanted to make sure requiring `dask-cudf` here was intentional 🙂